proto/prysm/v1alpha1/cloners.go

```diff
@@ -164,3 +164,15 @@ func CopyPayloadAttestationData(data *PayloadAttestationData) *PayloadAttestatio
 		PayloadStatus:   data.PayloadStatus,
 	}
 }
+
+// CopyPayloadAttestation copies the provided PayloadAttestation.
+func CopyPayloadAttestation(a *PayloadAttestation) *PayloadAttestation {
```

Comment:
It's not a big of a deal but this can replace L147-L152 with `newAttestations[i] = CopyPayloadAttestation(att)`.